src/benchq/compilation/jabalizer_utils.py

```diff
@@ -6,6 +6,22 @@
 from . import jl
 
 
+def create_adjlist_from_stabilizers(svec):
```

Comment:
The name of this function says "create_adjlist", but it actually creates a graph and not just adjacency list?
I think it should be renamed to `create_graph_from_stabilizers`?

Reply:
```suggestion
def create_graph_from_stabilizers(svec):
```